piweatherrock/plugin_weather_hourly/__init__.py

```diff
@@ -40,7 +40,10 @@ def disp_hourly(self, weather_rock):
             ampm = 'p.m.'
         this_hour_12_int = int(datetime.datetime.fromtimestamp(
             this_hour.time).strftime("%I"))
-        this_hour_string = "{} {}".format(str(this_hour_12_int), ampm)
+        if self.config["12hour_disp"]:
+            this_hour_string = "{} {}".format(str(this_hour_12_int), ampm)
+        else:
+            this_hour_string = "{} {}".format(str(this_hour_24_int), "hr")
```

Comment:
Does there really need to be a suffix? I ask because I am used to seeing 24h time w/o any suffix.

Reply:
In case of the hourly display, I'd leave it in. Otherwise it just say 21 22 23 0 above the icons. 21 hr 22 hr 23 hr 0 hr look better imo. On the main display the suffix could be left out perfectly

Reply:
Ah, makes sense